ql/src/java/org/apache/hadoop/hive/ql/exec/repl/util/ReplUtils.java

```diff
@@ -66,6 +66,10 @@
   // tasks.
   public static final String REPL_CURRENT_TBL_WRITE_ID = "hive.repl.current.table.write.id";
 
+  // Configuration to be received via WITH clause of REPL LOAD to rollback any previously failed
+  // bootstrap load.
+  public static final String REPL_ROLLBACK_BOOTSTRAP_LOAD_CONFIG = "hive.repl.rollback.bootstrap.load";
```

Comment:
Can this option be specified with a regular bootstrap directory?

Reply:
Nope. It works only with incremental dump.

Reply:
In that case, I think we need a better name. The current name suggests that it can be used for any bootstrap dump. I understand that this whole concept of incremental bootstrap is confusing. But we should try to reduce the confusion for the user.

Reply:
I want to keep the name generic to keep it simpler to support full bootstrap rollback in future. We can document it. Also, we throw error when they use full bootstrap to rollback.

Reply:
A bootstrap dump may contain objects other than table e.g. functions. We are not removing those when this option is specified. The current name is misleading in that sense as well.

Reply:
Ok. Do you have any suggestion?

Reply:
How about hive.repl.clean.tables.from.bootstrap? We can replace tables by relations if we want to includes materialized and normal view as well. Or other option is to keep the name as is but also code to remove other objects.

Reply:
Ok thanks! I like hive.repl.clean.tables.from.bootstrap. :)